DNN Platform/Library/Common/Utilities/NetworkUtils.cs

```diff
@@ -242,6 +243,24 @@ public static bool IsIPInRange(string currentIP, string startIP, string subnetma
             }
             return false;
         }
+
+        /// <summary>
+        /// Gets the Client IP address of the current request, from server variables if available, otherwise returns Request.UserHostAddress.
+        /// </summary>
+        /// <param name="request">The current http request.</param>
+        /// <returns>The current client ip address.</returns>
+        public static string GetClientIpAddress(HttpRequest request)
+        {
+            var ipAddress = request.ServerVariables["HTTP_X_FORWARDED_FOR"];
```

Comment:
Per the comment by @CMarius94 and confirmed with details linked here https://stackoverflow.com/questions/753645/how-do-i-get-the-correct-ip-from-http-x-forwarded-for-if-it-contains-multiple-ip it looks like we should split on a comma here and use the first value

Reply:
thanks for the review, updated the code